src/mm-progress-bar/mm-progress-bar.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.ProgressBar = Polymer({
+		is: 'mm-progress-bar',
+
+		behaviors: [
+			StrandTraits.Stylable
+		],
+
+		properties: {
+			ver: {
+				type: String,
+				value: '<<version>>'
+			},
```

Comment:
Don't need to include version strings anymore as we are planning on stamping a version into the "Strand" namespace